acceptable/util.py

```diff
@@ -50,7 +50,7 @@ def _sort_schema(schema):
             else:
                 yield v
     else:
-        yield d
+        yield schema
```

Comment:
Variable `d` doesn't exist. The only variable in this scope is `schema` and it does make sense to return it as is.